bin/polyply

```diff
@@ -81,6 +110,10 @@ def main():
     parser_gen_coords.add_argument('-name', required=True, type=str, dest="name",
                                    help="name of the final molecule")
 
+    parser_gen_coords.add_argument('-v', dest='verbosity', action='count',
+                             help='Enable debug logging output. Can be given '
+                                  'multiple times.', default=0)
+
```

Comment:
lint. And can't these be deduplicated?

Reply:
Not worth it considering the complexity of the parser already.